app/javascript/components/Pages/ListEmotions.js

```diff
@@ -80,6 +80,7 @@ function ListEmotions({ data,  setData , saveDataToDb, steps, service}) {
           <div className="mx-auto my-0 question">Which word best describes how you felt at work this week?</div>
             <div className='d-flex mx-auto emotions'>
                 {emotions.map((emotion, index) =>
+                  <div style={{width: 180}}>
```

Comment:
You should avoid using such 'style' implementation. Remove it pls and use class name with scss style